client/src/screens/week-availability.screen.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { Button, CheckBox, FlatList, StyleSheet, Text, View } from 'react-native';
+import { Divider, Icon, List } from 'react-native-elements';
```

Comment:
`List` is unused.
You might also find that `react-native-elements` is unnecessary if you take a look at some of the other screens. I have imported an icons library already and I believe there's another way of doing dividers in some of the other views. I'm not opposed to adding more deps if they're useful though. :)